src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT_PRICE = 40;
+  const DISCOUNT_OVER_6_DAYS = 50;
+  const DISCOUNT_OVER_2_DAYS = 20;
+  let basePrice = days * DAY_RENT_PRICE;
+
+  if (days > 6) {
+    return (basePrice -= DISCOUNT_OVER_6_DAYS);
+  }
+
+  if (days > 2) {
+    return (basePrice -= DISCOUNT_OVER_2_DAYS);
```

Comment:
The condition should be `days >= 3` to match the task requirement of a discount for rentals of 3 or more days. Similarly, return the calculated value without using assignment.